pkg/apis/extensions/v1alpha1/helper/dnsrecord_test.go

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package helper_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/utils/ptr"
+
+	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
+	. "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1/helper"
+)
+
+var _ = Describe("helper", func() {
```

Comment:
For consistency with the core API group helper tests:
```suggestion
var _ = Describe("Helper", func() {
```